docs/create/azure/index-azure.mdx

```diff
@@ -1,52 +1,29 @@
 ---
 id: index-azure
-title: Create Redis database on Azure Cache
-sidebar_label: Redis on Azure Cache
+title: Azure Cache for Redis
+sidebar_label: Overview
 slug: /create/azure
 ---
 
-Azure Cache for Redis is a native fully-managed service on Microsoft Azure. Azure Cache for Redis offers both the Redis open-source (OSS Redis) and a commercial product from Redis Labs (Redis Enterprise) as a managed service. It provides secure and dedicated Redis server instances and full Redis API compatibility. The service is operated by Microsoft, hosted on Azure, and accessible to any application within or outside of Azure.
+import RedisCard from '@site/src/theme/RedisCard';
 
+The following links provides you with the available options for create instances of Azure Cache for Redis
 
+<div class="row">
 
-### Step 1. Getting Started 
+    <div class="col">
```

Comment:
![image](https://user-images.githubusercontent.com/42971704/125650654-042a01e8-cb05-41ac-a62e-b135be60cf82.png)
this doesn't seem to be coming through quite right.

Reply:
Fixed.

Reply:
Fixed.